apps/design-land/src/app/core/code-preview/model/code-example.ts

```diff
@@ -0,0 +1,15 @@
+export interface DesignLandCodeExampleFile {
+  name: string;
+  content: string;
+  language: string;
+}
+
+export interface DesignLandCodeExample {
+  /**
+   * The custom element to render in the dom.
+   * E.g. <my-custom-element></my-custom-element>
+   */
+  element: string;
+
+  files: DesignLandCodeExampleFile[];
```

Comment:
Needs docs.

Reply:
Addressed in 4ca1a7a6ebc5f087068f9667d324b57ecc82e893